scripts/migrateHistoricBids.ts

```diff
@@ -0,0 +1,130 @@
+import * as fs from "fs";
+import * as env from "env-var";
+import * as dotenv from "dotenv";
+import * as mongodb from "mongodb";
+import { Bid } from "../src/types";
+import { MongoClientOptions } from "mongodb";
+import { queueAdapters, MessageQueueService } from "../src/messagequeue";
+import {
+  MessageType,
+  TypedMessage,
+  BidPlacedV1Data,
+  BidCancelledV1Data,
+} from "@zero-tech/zns-message-schemas";
+dotenv.config();
+
+const argv = require("yargs").argv;
+const outputFilename = "./scripts/output/zAuctionBidsHistorical.json";
+const user = env.get("MONGO_USERNAME").required().asString();
+const pass = env.get("MONGO_PASSWORD").required().asString();
+const uri = env.get("MONGO_CLUSTER_URI").required().asString();
+const dbName = env.get("MONGO_DB").required().asString();
+const collectionName = env.get("MONGO_COLLECTION").required().asString();
+const fullUri = `mongodb+srv://${user}:${pass}@${uri}/`;
+const options: MongoClientOptions = {
+  connectTimeoutMS: 5000,
+  w: "majority",
+};
+
+const main = async () => {
+  const client = new mongodb.MongoClient(fullUri, options);
+  await client.connect();
+  const database = client.db(dbName);
+  const bidsPlaced = await getAllFromCollection<Bid>(collectionName, database);
```

Comment:
Missed this in my first review pass. bidPlaced messages need to be created from the bid collection AND the bid-archive collection. This is b/c bid is deleted from the `bid` collection once it's moved to a `bid-archive`.

Reply:
Added, its now pulling from archive twice: Once as BidAccepted, another as BidCancelled